src/sidebar/util/time.js

```diff
@@ -151,40 +147,47 @@ const BREAKPOINTS = [
  *
  * @param {Date} date - The date to consider as the timestamp to format.
  * @param {Date} now - The date to consider as the current time.
- * @return {breakpoint} A dict that describes how to format the date.
+ * @return {breakpoint|null} An object that describes how to format the date or
+ *                           null if no breakpoint matches.
  */
 function getBreakpoint(date, now) {
-  let breakpoint;
-  for (let i = 0; i < BREAKPOINTS.length; i++) {
-    breakpoint = BREAKPOINTS[i];
+  for (let breakpoint of BREAKPOINTS) {
     if (breakpoint.test(date, now)) {
       return breakpoint;
     }
   }
-
   return null;
 }
 
+/**
+ * Return the number of milliseconds until the next update for a given date
+ * should be handled, based on the delta between `date` and `now`.
+ *
+ * @param {Date} date
+ * @param {Date} now
+ * @return {Number|null} - ms until next update or `null` if no update
+ *                         should occur
+ */
 function nextFuzzyUpdate(date, now) {
   if (!date) {
     return null;
   }
 
-  let secs = getBreakpoint(date, now).nextUpdate;
+  let nextUpdate = getBreakpoint(date, now).nextUpdate;
```

Comment:
Not in seconds anymore; renamed to something more generic.

Reply:
Yes: All things in the module are `ms`. They used to be a mix of seconds and ms.